radixdlt/src/main/java/com/radixdlt/consensus/Vertex.java

```diff
@@ -161,12 +160,12 @@ private Long getSerializerView() {
 
 	@Override
 	public String toString() {
-		return String.format("Vertex{epoch=%s view=%s, qc=%s, atom=%s}", epoch, view, qc, atom == null ? null : atom.getAID());
+		return String.format("Vertex{epoch=%s view=%s, qc=%s, cmd=%s}", epoch, view, qc, command);
```

Comment:
This means that class Command should have some sort of sensible toString() method.